README.md

```
@@ -1,33 +1,42 @@
 
-<!-- Message passing: *"Do not communicate by sharing memory; instead, share memory by communicating."* -->
+This feature will be available soon.
+
+## Usage
+
+An example of how to use the library is provided in the `examples` directory.
+- `example1.rs` demonstrates how to randomly create a network, train it to memorize a random spike train, and simulate the network.
 
 ## References
 
 * [Aguettaz and Loeliger, "Continuous-time neural networks can stably memorize random spike trains", *arXiv*, 2024.](https://arxiv.org/abs/2408.01166)
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)